src/SyncStatusOnPostUpdate.php

```diff
@@ -40,6 +40,11 @@ public function on_post_status_change( $new_status, $old_status, $post ) {
 			Fns::map( Obj::prop( 'element_id' ) )
 		);
 
+		// Prevent running logic if the old and new status are the same
+		if ( $new_status == $old_status ) {
```

Comment:
@GeorgeBotsev, can you use a strict comparison `$new_status == $old_status`? (these variables should be strings)

Reply:
Okay, I will add that.

Reply:
Resolved in commit #1 ceda2d5